verilog/tools/kythe/kythe_facts.h

```diff
@@ -101,6 +104,50 @@ struct VName {
 
 std::ostream& operator<<(std::ostream&, const VName&);
 
+// Facts for kythe.
+struct Fact {
+  Fact(const VName& vname, absl::string_view name, absl::string_view value)
+      : node_vname(vname), fact_name(name), fact_value(value) {}
+
+  bool operator==(const Fact& o) const;
```

Comment:
The argument name in the header and definition should either match, or if it is obvious, can be omitted in the header.